### ISSUE TITLE
ContextPart not available in Pharo 5

### ISSUE THREAD

**Issue by Uko** [SeasideSt/Seaside#868]
When loading Seaside (Seaside-Pharo-Continuation) into Pharo 5, you get a message

```
This package depends on the following classes:
  ContextPart
You must resolve these dependencies before you will be able to load these definitions: 
  ContextPart>>#localAt:
  ContextPart>>#localAt:put:
  ContextPart>>#localSize
  ContextPart>>#serialiseOn:
```

**Comment by jbrichau** [comment]
Hi @Uko 

This was solved for Seaside 3.2 but, intentionally, not for Seaside 3.1.
Are you loading Seaside 3.1.0 in Pharo5? If that's the case, do you have a specific reason for not moving to 3.2 ?

**Comment by marschall** [comment]
The issue is that the configuration in the meta repo is outdated. I opened for this #876
